package.json

```diff
@@ -1 +1,77 @@
-{"extensionDependencies":["ms-azuretools.vscode-docker"],"dependencies":{"@microsoft/vscode-container-client":"^0.1.0","vscode-jsonrpc":"^8.1.0"},"name":"securesupplychain","displayName":"securesupplychain","description":"securesupplychain features","version":"0.0.1","engines":{"vscode":"^1.80.0"},"categories":["Other"],"activationEvents":[],"main":"./out/extension.js","contributes":{"commands":[{"command":"securesupplychain.showReferrers","title":"Show Referrers"}],"menus":{"view/item/context":[{"command":"securesupplychain.showReferrers","when":"view == dockerRegistries && viewItem == azure;DockerV2;Tag;","group":"regs_tag_1_general@7"}]}},"scripts":{"vscode:prepublish":"npm run compile","compile":"tsc -p ./","watch":"tsc -watch -p ./","pretest":"npm run compile && npm run lint","lint":"eslint src --ext ts","test":"node ./out/test/runTest.js"},"devDependencies":{"@types/glob":"^8.1.0","@types/mocha":"^10.0.1","@types/node":"20.2.5","@types/vscode":"^1.80.0","@typescript-eslint/eslint-plugin":"^5.59.8","@typescript-eslint/parser":"^5.59.8","@vscode/test-electron":"^2.3.2","eslint":"^8.41.0","glob":"^8.1.0","mocha":"^10.2.0","typescript":"^5.1.3"}}
\ No newline at end of file
+{
+    "extensionDependencies": [
+      "ms-azuretools.vscode-docker"
+    ],
+    "dependencies": {
+      "@microsoft/vscode-container-client": "^0.1.0",
+      "vscode-jsonrpc": "^8.1.0"
+    },
+    "name": "securesupplychain",
+    "displayName": "securesupplychain",
+    "description": "securesupplychain features",
+    "version": "0.0.1",
+    "engines": {
+      "vscode": "^1.80.0"
+    },
+    "categories": [
+      "Other"
+    ],
+    "activationEvents": [],
+    "main": "./out/extension.js",
+    "contributes": {
+      "commands": [
+        {
+          "command": "securesupplychain.showReferrers",
+          "title": "Show Referrers"
+        },
+        {
+          "command": "securesupplychain.defaultKey",
```

Comment:
It seems weird that a command name has noun for a name.  Why not 'securesupplychain.signImage'?